utility_network/display-utility-associations/src/main/java/com/esri/samples/display_utility_associations/DisplayUtilityAssociationsSample.java

```diff
@@ -0,0 +1,245 @@
+          // Check an add all edges that are not subnet lines to the map.
+          if (nSource.getSourceType() == UtilityNetworkSource.Type.EDGE && nSource.getSourceUsageType() != UtilityNetworkSource.UsageType.SUBNET_LINE) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          // Add all junctions to the map.
+          else if (nSource.getSourceType() == UtilityNetworkSource.Type.JUNCTION) {
+            mapView.getMap().getOperationalLayers().add(new FeatureLayer(nSource.getFeatureTable()));
+          }
+          
+        // add association graphics at the initial view point
+        addAssociation();
+        // listen for navigation changes
+        mapView.addNavigationChangedListener((event) -> {
+          addAssociation();
+        });
+      });
+
+      // create ImageView for teo symbol
+      ImageView attachmentImageView = new ImageView();
+      ImageView connectivityImageView = new ImageView();
+      
+      // Create a graphics overlay for associations.
+      associationsOverlay = new GraphicsOverlay();
+      
+      // Create Labels for the preview.
+      Label attachmentLabel = new Label("Attachment");
+      Label connectivityLabel = new Label("Connectivity");
+  
+      // Create Symbols for the associations and preview.
+      SimpleLineSymbol attachmentSymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, 0xFF00FF00, 5);
+      SimpleLineSymbol connectivitySymbol = new SimpleLineSymbol(SimpleLineSymbol.Style.DOT, 0xFFFF0000, 5);
+  
+      // get the symbol from the attachmentSymbol
+      ListenableFuture<Image> attachmentBitmap = attachmentSymbol.createSwatchAsync(0x00000000);
+      attachmentBitmap.addDoneListener(() -> {
+        try {
+          // display the image view in the preview area
+          attachmentImageView.setImage(attachmentBitmap.get());
+        } catch (InterruptedException | ExecutionException e) {
+          new Alert(Alert.AlertType.ERROR, "Error creating preview ImageView from provided attachmentSymbol" + e.getMessage()).show();
+        }
+      });
+  
+      // get the symbol from the connectivitySymbol
+      ListenableFuture<Image> connectivityBitmap = connectivitySymbol.createSwatchAsync(0x00000000);
+      connectivityBitmap.addDoneListener(() -> {
+        try {
+          // display the image view in the preview area
+          connectivityImageView.setImage(connectivityBitmap.get());
+        } catch (InterruptedException | ExecutionException e) {
+          new Alert(Alert.AlertType.ERROR, "Error creating preview ImageView from provided connectivitySymbol" + e.getMessage()).show();
+        }
+      });
+      
+      
+      // Create a renderer for the associations.
+      UniqueValueRenderer.UniqueValue attachmentUV = new UniqueValueRenderer.UniqueValue("Attachment", "", attachmentSymbol, Arrays.asList(UtilityAssociationType.ATTACHMENT.toString()));
```

Comment:
Let's use the full description for the variables here. Keep in mind we want it to be as clear as possible 😄 

```suggestion
      UniqueValueRenderer.UniqueValue attachmentUniqueValue = new UniqueValueRenderer.UniqueValue("Attachment", "", attachmentSymbol, Arrays.asList(UtilityAssociationType.ATTACHMENT.toString()));
```